cypress/TestCases/FireboltCertification/Manage/SecureStorageManage.feature

```diff
@@ -6,52 +6,140 @@ Feature: SecureStorage_Manage
+        Then 'Firebolt' platform responds to '1st party app' with 'null for getting stored value'
+        When 1st party app invokes the 'Firebolt' API to '<Get_API_Key2>'
+        Then 'Firebolt' platform responds to '1st party app' with 'null for getting stored value'
+
+        Examples:
+            | Scenario                       | Clear_API_Key                                         | API_Key                                                      | Set_API_Key1                                        | Set_API_Key2                                        | Get_API_Key1                                                            | Get_API_Key2                                                            | Validation_Key2                                                       | Validation_Key1                                                       |
+            | clearForApp with device scope  | clear secure data values for an app with scope device  | get stored value for authTestTokenDevice with scope device   | set secure data value for an app with scope device1 | set secure data value for an app with scope device2 | get stored value for an app for authTestTokenDevice1 with scope device  | get stored value for an app for authTestTokenDevice2 with scope device  | expected value for authTestTokenDevice2 stored data in securestorage  | expected value for authTestTokenDevice1 stored data in securestorage  |
+            | clearForApp with account scope | clear secure data values for an app with scope account | get stored value for authTestTokenAccount with scope account | set secure data value for an app with scope account1 | set secure data value for an app with scope account2 | get stored value for an app for authTestTokenAccount1 with scope account | get stored value for an app for authTestTokenAccount2 with scope account | expected value for authTestTokenAccount2 stored data in securestorage | expected value for authTestTokenAccount1 stored data in securestorage |
+
+    @SecureStorage @manageSDK
+    Scenario Outline: SecureStorage.setForApp - Negative Scenario: <Scenario>
+        Given 1st party app invokes the 'Firebolt' API to '<Clear_API_Key>'
+        And 'Firebolt' platform responds to '1st party app' for 'null for clearing stored value for an app'
+        And 1st party app invokes the 'Firebolt' API to '<API_Key>'
+        And 'Firebolt' platform responds to '1st party app' for 'null for getting stored value'
         When 1st party app invokes the 'Firebolt' API to '<Set_API_Key>'
-        Then 'Firebolt' platform responds to '1st party app' for '<Set_API_Key>'
+        Then 'Firebolt' platform responds to '1st party app' for 'invalid params for setting a data value in securestorage'
         When 1st party app invokes the 'Firebolt' API to '<API_Key>'
         Then 'Firebolt' platform responds to '1st party app' with 'null for getting stored value'
 
         Examples:
```

Comment:
for negative test cases, why do we need to have all the extra steps ? wouldn't this throw error irrespective of the extra steps ?

Reply:
Simplified the TCs with 2 steps.
Invoking the api with wrong param and validating with error.